sdk/spring/spring-cloud-azure-stream-binder-eventhubs/CHANGELOG.md

```diff
@@ -3,12 +3,14 @@
 ## 4.0.0-beta.1 (Unreleased)
 
 ### Features Added
-
+- Support batch consumers.
 ### Breaking Changes
 - Change artifact id from `azure-spring-cloud-stream-binder-eventhubs` to `spring-cloud-azure-stream-binder-eventhubs`.
 - Change the binder type from `eventhub` to `eventhubs`.
 - Change the Spring Cloud Stream Binding extended properties prefix from `spring.cloud.stream.eventhub` to `spring.cloud.stream.eventhubs`. 
 
+- When consume batch mode is set to `true`, then checkpoint mode only support `BATCH` OR `MANUAL`; While consume 
```

Comment:
I think we could simply this point like: BATCH checkpoint-mode only works in batch-consuming mode.

Reply:
updated